sapl-spring-data-mongo-reactive/src/test/java/io/sapl/springdatamongoreactive/sapl/queries/enforcement/MongoAnnotationQueryManipulationEnforcementPointTest.java

```diff
@@ -87,7 +88,7 @@ class MongoAnnotationQueryManipulationEnforcementPointTest {
 
     @BeforeAll
     public static void setUp() throws JsonProcessingException {
-        obligations            = objectMapper.readTree(
+        obligations            = (ArrayNode) objectMapper.readTree(
```

Comment:
use variant with ArrayNode.class as parameter ?

Reply:
Can't find variant with ArrayNode.class as parameter. Or I'm too dumb to find it.

Reply:
Something like this ?

```java
  var expectedResource = MAPPER.readValue("""
                {
                   "last":"5"
                }
                """, JsonNode.class);
```

Reply:
Very nice, works fine, thank you